docs/Operation-Architecture/Orbit_UI/File_Operations/File_Operations.rst

```diff
@@ -1,14 +1,14 @@
 File Operations
 ===============
 
-**Prerequisites:** You must have at least one account, containing at
+**Prerequisites:** You must have at least one account, containing at
```

Comment:
is this supposed to be in a `note` or `important` box?

Reply:
No.